### ISSUE TITLE
xmrstak difficulty set to 600023 with Nanopool

### ISSUE THREAD

**Issue by Theflyer737** [fireice-uk/xmr-stak#2535]
Hello,

When starting Xmrstak since 2 days, for Nanopool, difficulty is presented as set to 600023.
Usually the difficulty for this pool was 120000.

Threadripper 1950x
Nvidia Geforce 1060 and Geforce 1070Ti

I never had this issue before and xmrstak was running quickly and correctly at difficulty 120000.

Are ASICS back again on the network?

Rgds

**Comment by Ryo-RU** [comment]
might be pool vardiff issue.
try another pool, set static diff to 120k and see if it increases also.

**Comment by Theflyer737** [comment]
I tried another pool and difficulty is set to normal level.

Where can I set the static diff ? 

 

De : Ryo-RU <notifications@github.com> 
Envoyé : mardi 22 octobre 2019 11:52
À : fireice-uk/xmr-stak <xmr-stak@noreply.github.com>
Cc : Theflyer737 <fa053728@skynet.be>; Author <author@noreply.github.com>
Objet : Re: [fireice-uk/xmr-stak] xmrstak difficulty set to 600023 with Nanopool (#2535)

 

might be pool vardiff issue.
try another pool, set static diff to 120k and see if it increases also.
